docs/2_2_common_mechanisms.adoc

```diff
@@ -574,7 +574,8 @@ The following Clock types are described in detail after the table.
 Input Clock::
 [[inputClock, `input Clock`]]
 is a variable of type Clock with <<causality>> == <<input>>.
-The importer will activate input Clocks using <<fmi3SetClock>> or <<fmi3ActivateModelPartition>>.
+In Model Exchange and Co-Simulation, when input Clocks tick, the importer will activate the input Clocks using <<fmi3SetClock>>.
+In Scheduled Execution, when input Clocks tick, the importer will activate the associated model partitions (but not the input Clocks) using <<fmi3ActivateModelPartition>>.
```

Comment:
I think it is the other way round, when a model partition is activated, the clock ticks ...

Reply:
I'd say the clock tick determines the time instant of the model partition's activation and is causally first. E.g. a triggered input clock might tick because a triggered output clock of another FMU ticked and the two are connected; the tick of the input clock then clearly causes and precedes the activation of the model partition.